### PR TITLE
Allow more click modes

### PR DESCRIPTION
**Pull request by kaduvert**

### CODE REVIEW

**Comment by kaduvert** [comment]
this may be incomplete because i don't know how the information sent in the packets differs between versions
in 1.8 it sends null item with mode 2 or 4

File: lib/plugins/inventory.js [hunk 1]
```diff
@@ -528,7 +527,7 @@ function inject (bot, { hideErrors }) {
         mouseButton,
         action: actionId,
         mode,
-        item: Item.toNotch(click.item)
```
Comment (rom1504) [inline lib/plugins/inventory.js:531]: Could you add a comment explaining this line ?

Comment (kaduvert) [reply lib/plugins/inventory.js:531]: in 1.8 it sends null as item with mode 2 or 4
i assumed this is up to 1.16.5 but that may not be correct, haven't tested

Comment (rom1504) [reply lib/plugins/inventory.js:531]: Ok can you add a comment for this ?

Comment (kaduvert) [reply lib/plugins/inventory.js:531]: yes, did it

at first i thought you meant that i should explain it to you, my bad

**Comment by rom1504** [comment]
would be good to add tests but lgtm

**Pull request merged**